drivers/lte_link_control/Kconfig

```diff
@@ -83,6 +83,19 @@ config LTE_PDP_CONTEXT
 		The +CGDCONT command defines Packet Data Protocol (PDP) Context.
 		For reference, see 3GPP 27.007 Ch. 10.1.1
 
+choice
+	prompt "Select network mode"
+	default LTE_NETWORK_MODE_LTE
+	help
+		Select either LTE or Narrowband-IOT (NB_IOT) network mode
```

Comment:
`NB_IOT` --> `NB-IoT` to conform with other NCS docs
As mentioned in a previous comment, I think it would be clearer to use "LTE-M" or M1 instead of only "LTE".

Reply:
@ru-fu Do you have any input on the naming here to align with the rest of NCS language?

Reply:
I agree: LTE-M and NB-IoT